modules/administration/pages/hostname-rename.adoc

```diff
@@ -0,0 +1,36 @@
+[[hostname-rename]]
+= Rename {productname} Server
+
+It is possible to rename a {productname} Server.
+Begin by changing the network settings of the server locally and remotely.
+Then use the [command]``spacewalk-hostname-rename`` script to update the settings in the PostgreSQL database and the internal structures of {productname}.
+
+The [command]``spacewalk-hostname-rename`` script is part of the [package]``spacewalk-utils`` package.
+
+The only mandatory parameter for the script is the newly configured IP address of the {productname} Server.
+
+
+
+.Procedure: Renaming {productname} Server
+. Change the network settings of the server on the system level locally and remotely at the DNS server.
+Provide configuration settings for reverse name resolution, too.
+Changing network settings is done in the same way as with renaming any other system.
+. Restart your network configuration to make the {productname} Server uses the new hostname.
+. Reboot the {productname} Server to ensure the hostname has changed.
+If the server is not using the new name, the script will fail.
```

Comment:
Shouldn't this a note?

Reply:
Within procedure, we no longer want admonitions such as formal notes.  Maybe, we can fall back to fake notes, starting with `NOTE:` on a new line.

Reply:
alternatively, you could do it as a proper admonition at the end of the procedure? @jcayouette this is probably an important thing to consider when you are talking to Naman about block styling, btw.

Reply:
for  a simple NOTE it is probably ok to have it at the end of the procedure.